homeassistant/components/snmp/switch.py

```diff
@@ -240,7 +240,7 @@ async def _execute_command(self, command):
             await self._set(command)
         # User set vartype Null, command must be an empty string
         elif self._vartype == "Null":
-            await self._set(Null)("")
+            await self._set(Null)
```

Comment:
Not 100% sure this is correct, but do not think `self._set` returns a callable

Reply:
@Mich-b any comment here?

Reply:
Sorry, don't remember this part. Did you test it?

Reply:
Not sure about this either, but to be safe you could also revert the change and just add pylint disable here
```suggestion
            await self._set(Null)("")  # pylint: disable=not-callable
```

Reply:
No, able to.

Reply:
Hmm, does not feel good to disable a test for something that is obvious incorrect :/

Reply:
I would suggest `await self._set("")` since that is how I understand the comment:
> #User set vartype Null, command must be an empty string

Reply:
It certainly seems like that was what was intended here

Reply:
```suggestion
            await self._set("")
```